test/integration/lib/philipsHue_server.js

```diff
@@ -160,9 +104,112 @@ var philipsHue_server = (function() {
      });
    });
   }
+   
+  // Callback methods
+
+  _server.use(function (req, res, next) { 
+    // foxbox does not have the json header, so need to 
+    // inject it here so the message body can be properly captured
+    req.headers['content-type'] = 'application/json';
+    last_received_cmd = req.method + ':' + req.originalUrl;
+    next();
+  });
+
+  _server.get('/', function (req, res) {
+    res.status(200).sendFile(
+      path.join(__dirname,'/html/philips_initial.html'));  
+  });
+
+  _server.get('/api/:foxboxId/', function (req, res) {   
+      
+     if (foxboxId === undefined) {
+      if (auth_status === false) {
+        foxboxId = req.params.foxboxId;
+        res.sendStatus(200);  
+      }
+      else if (auth_status === true) {
+        res.status(200).send([{
```

Comment:
Okay, in this case, just add a comment saying we mimic philips hue, but we know it's weird